src/palace.pyx

```diff
@@ -133,21 +143,27 @@ def current_context() -> Optional[Context]:
     return current
 
 
-def use_context(context: Optional[Context]) -> None:
+def use_context(context: Optional[Context], thread: bool = False) -> None:
     """Make the specified context current for OpenAL operations.
 
+    If `thread` is set to `True`,
+    make the specified context current for OpenAL operations on the calling
+    thread only.  Require the `ALC_EXT_thread_local_context` extension on
+    both the context's device and `devmgr`.  You can use `query_extension`
+    and `device.query_extension` to check for that.
```

Comment:
Same as above.

Reply:
When I say same as above, I did not meant to do it literally the same.  `use_context` does not return anything.  Also please remove the *co-author* thing when you squash, since the squashed commit will be 99% your work.

Reply:
What do you mean by co-author thing?

Reply:
See the commit message of f37d2a479dde49474174bb5b852df3cc0db25120.

Reply:
When you squash later, mind the commit message and only leave what's really useful.